runtime/runtime_test.go

```diff
@@ -438,7 +438,7 @@ func (i *testRuntimeInterface) ImplementationDebugLog(message string) error {
 
 func (i *testRuntimeInterface) ValidatePublicKey(key *PublicKey) (bool, error) {
 	if i.validatePublicKey == nil {
-		return false, nil
+		return true, nil
```

Comment:
The default implementation for all mock functions is to fail. If a successful result is expected, I'd recommend passing a `validatePublicKey` function in, i.e. `alwaysValidates` which got introduced above